reduxApp2/dev/js/components/App.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import FillDetails from '../containers/fill-details';
+import UserDetails from '../containers/user-details';
```

Comment:
@VARTIKASINGH1234 Instead of 'UserDetails' it should be like DisplayUserDetails.

Reply:
I have made this change in next file